marm-mcp-server/marm_mcp_server/server.py

```
@@ -210,35 +262,83 @@ async def _mcp_tool_call_tracker(request: Request, call_next):
+                        {
+                            "type": "text",
+                            "text": f"[MARM SESSION INIT]\n\n{protocol_content}",
+                        }
+                    )
+                    _protocol_delivered = True
+                    protocol_injected = True
+
+            _req_session = None
+            try:
+                _req_session = json.loads(body).get("params", {}).get("arguments", {}).get("session_name")
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)